__tests__/get-endpoints.test.js

```diff
@@ -80,13 +80,86 @@ describe("GET /api/articles/:id", () => {
                 expect(result.body).toEqual({ message: "Article not found" });
             });
     });
+
+
+    test("Handles invalid endpoint", () => {
+        return request(app)
+            .get("/api/reviews/banana")
+            .then((result) => {
+                expect(result.status).toBe(404);
+                expect(result.body).toEqual({ message: "Not found" });
+            });
+    });
 });
 
+describe("GET /api/articles", () => {
+    test("responds with an array of article objects", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((result) => {
+                expect(Array.isArray(result.body)).toBe(true);
+                expect(Object.keys(result.body[0])).toHaveLength(9);
+                expect(result.body[0]).toEqual({
```

Comment:
Ideally we should be handling this assertion with a forEach and asserting that each element of the array looks a certain way. expect.any() for the value can be really useful here. You'll find this will make these tests easier to write and give you confidence that everything inside the array looks as expected rather than just the first few.
We also need an assertion for checking the results come back in the correct order. Jest sorted can help with that :)